keyform/views.py

```diff
@@ -25,11 +27,55 @@ def get_queryset(self):
 
 class RequestView(LoginRequiredMixin, UpdateView):
     model = Request
+    success_url = reverse_lazy('home')
     template_name = "keyform/request.html"
     form_class = EditForm
 
     def get_success_url(self):
-        return reverse('home')
+        return reverse_lazy('home')
```

Comment:
You don't need to define a `get_success_url()` method if you have the `success_url` attribute already defined.